shell.nix

```diff
@@ -4,7 +4,7 @@
   # the bky-as cli the example test works against is updated.
   #
   # This default value can be overwritten from the command line.
-  bkyAsVersion ? "e7a2c061da429c66dcfadaf6007e0a4161fea6dc",
+  bkyAsVersion ? "latest",
```

Comment:
I updated to use latest but I am not sure if that is a great idea.... although it was very cool when a PR was merged to delphi while I was working on this.  I created a new shell and everything just updated!  It was kind of magical. :)